picasso/src/main/java/com/squareup/picasso3/BitmapHunter.java

```diff
@@ -115,11 +113,11 @@
       dispatcher.dispatchRetry(this);
     } catch (OutOfMemoryError e) {
       Buffer buffer = new Buffer();
-      try {
-        stats.createSnapshot().dump(buffer);
-      } catch (IOException ioe) {
-        throw new AssertionError(ioe);
-      }
+      //try {
+      //  //picasso.createSnapshot().dump(buffer);
+      //} catch (IOException ioe) {
+      //  throw new AssertionError(ioe);
+      //}
```

Comment:
not sure what to do here.  thoughts?

Reply:
Just remove it. We should be propagating the OOM directly, in actuality. Not sure whether we're set up for that though...

Reply:
removed the OOM catch clause (can figure out a better strategy later).